Pod/Classes/HairlineView/README.md

```diff
@@ -0,0 +1,48 @@
+# HairlineView
+
+A UIView that can be defined by axis, thickness and color
+
+## Default Appearance
+
+<details>
+<summary>Screenshots</summary>
+
+<p>
+<img src="HairlineTableCells.png" width="200">
+</p>
+
+</details>
+
+### Quick Start
+
+A simple 1 pixel, horizontal, darkGray hairline:
+```swift
+let hairline = HairlineView()
+cell.contentView.addSubview(hairline)
+hairline.translatesAutoresizingMaskIntoConstraints = false
+hairline.widthAnchor.constraint(equalTo: cell.contentView.widthAnchor).isActive = true
+hairline.bottomAnchor.constraint(equalTo: cell.contentView.bottomAnchor).isActive = true
```

Comment:
This hairline will be horizontally ambiguous - needs a constraint for x-position. Also, would be nice to have a comment about how you don't have to give it a constraint for height because it defaults to horizontal.

Reply:
nice catch; copy paste error